Remove conflicting definitions of system functions and variables. The system
headers already provide the correct ones.

https://github.com/DALnet/bahamut/issues/230
--- src/res.c.orig	2022-08-16 10:40:51.000000000 -0500
+++ src/res.c	2023-07-06 14:50:35.000000000 -0500
@@ -58,15 +58,6 @@
 
 #define RES_HOSTLEN 127 /* big enough to handle addresses in in6.arpa */
 
-extern int  dn_expand(char *, char *, char *, char *, int);
-extern int  dn_skipname(char *, char *);
-extern int
-res_mkquery(int, char *, int, int, char *, int,
-	    struct rrec *, char *, int);
-
-#ifndef AIX
-extern int  errno, h_errno;
-#endif
 extern int  highest_fd;
 extern aClient *local[];
 
